bids_prov/afni/afni_parser.py

```diff
@@ -44,7 +44,7 @@
 )
 
 
-def clean_label_suffix(label:str) ->str:
+def clean_label_suffix(label: str) -> str:
     """ Erase suffix like tlrc in label to keep link of passed entities label
```

Comment:
more cleaning instructions in other branch dev-afni (PR 99). To be merged after

Reply:
Done with the merge